packages/node/src/utils/generate-package-json.ts

```diff
@@ -45,6 +45,16 @@ export function generatePackageJson(
     packageJson.dependencies[packageName] = version;
   });
 
+  options.implicitDependencies.forEach((packageName) => {
+    const version = rootPackageJson.dependencies[packageName];
```

Comment:
Yes, but that case in handled in the line 51, if the required dependency not exist in the root package.json, that dependency will not be included in the generated json... If the app need `rxjs`, the rxjs package need to be installed in the workspace. Why not pass the package name and version in the builder options?, to avoid having multiple apps with different versions of the same library, the root package.json should be the source of truth